utils/box-symbol.stanza

```diff
@@ -244,6 +244,24 @@ public defn make-box-symbol (banks:Seqable<BoxBank>) :
 ;=======================================================<doc>
 
 public defn make-box-symbol () :
+  ;Sort the given pins by their .pin-properties-row-index field.
+  defn sort-by-row-index (pins:Tuple<JITXObject>) -> Tuple<JITXObject> :
+    ;Compare pin[i0] versus pin[i1].
+    ;Indices are used, instead of the pin directly, in order to perform a
+    ;stable sort.
+    ;All pins without a .pin-properties-row-index are put in the beginning.
+    defn compare-index (i0:Int, i1:Int) :
+      val a = pins[i0]
+      val b = pins[i1]
```

Comment:
minor but you does not really need these two lines

Reply:
You're right. I would like it to be inlined too.